epsagon/handler.py

```diff
@@ -0,0 +1,29 @@
+"""
+Epsagon generic wrapper used only in Lambda environments.
+"""
+
+from .utils import init, import_original_module
+from .wrappers import lambda_wrapper
+
+init()
```

Comment:
think about moving the environment variables reading to here. not sure what makes more sense

Reply:
I prefer to leave them on the init, because they are relevant even if not using the generic handler.